waku/v2/node/config.nim

```diff
@@ -247,6 +247,21 @@ type
       desc: "WebSocket listening port."
       defaultValue: 8000
       name: "websocket-port" }: Port
+    
+    websocketSecureSupport* {.
+      desc: "Enable secure websocket:  true|false",
+      defaultValue: false
+      name: "websocket-secure-support"}: bool
+    
+    websocketSecureKey* {.
+      desc: "Secure websocket key path:   '/path/to/key.txt' ",
+      defaultValue: ""
+      name: "websocket-secure-key"}: string
+    
+    websocketSecureCert* {.
```

Comment:
What do you think about changing it to `websocketSecureCertPath` to be more accurate? 
ditto `websocketSecureKey`